TicTacToe2/TicTacToe/Player.cs

```diff
@@ -6,22 +6,30 @@ namespace TicTacToe2.TicTacToe
 {
     class Player
     {
-        /*public Player(char symbol)
-        {
 
+        public Player(char symbol)
+        {
+            Symbol = symbol;
         }
 
         public void GetMove(out int x, out int y)  //we need to return multiple values
         {
-
+            x = 0;
+            y = 0;
         }
 
         public char GetSymbol()
```

Comment:
I can't remember if I said to create a GetSymbol function as part of the interface, but this is redundant with the Symbol property. We can already pass X or O to the constructor when creating an object of type Player, so GetSymbol can be reduced to
```c#
return Symbol;
```
The Symbol property is already public, so this is not needed.